apps/point_cloud_editor/include/pcl/apps/point_cloud_editor/selection.h

```diff
@@ -63,7 +63,9 @@ class Selection : public Statistics
     /// @brief Copy constructor
     /// @param copy The selection object to be copied
     Selection (const Selection& copy)
-      : cloud_ptr_(copy.cloud_ptr_), selected_indices_(copy.selected_indices_)
+    : Statistics (copy)
+    , cloud_ptr_(copy.cloud_ptr_)
+    , selected_indices_(copy.selected_indices_)
```

Comment:
This looks like an explicitly written default copy ctor. It can be deleted.

Reply:
Again, according to [wiki](https://en.wikipedia.org/wiki/Special_member_functions) copy-constructor won't be generated if destructor is declared (which it is). So what we can do is either default this one and keep destructor, or (better) remove destructor (it's empty anyway) and this one (let compiler generate it).

Reply:
I like the option of deleting both. Less code to maintain :D